AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt

```diff
@@ -8,6 +8,10 @@
 ! @@||cdn.cookielaw.org/scripttemplates/otSDKStub.js
 @@||cdn.cookielaw.org/scripttemplates/otSDKStub.js$domain=nintendo.*|mtv.com|ryobitools.eu|cc.com|3sat.de|zdf.de|costco.com|giantbomb.com|glovoapp.com|rte.ie|digitaltrends.com|spiele.bild.de|spiele.stern.de|vanityfair.com|saarbruecker-zeitung.de|olympics.com|tvn.pl|bergfreunde.de|oe24.at|epicgames.com|toteraz.pl|natgeotv.com|midomi.com|volksfreund.de|kawasaki.com|foxchannels.com.tr|ga.de|shop.mango.com|kralmuzik.com.tr|westernunion.com|roastmarket.de|new.cappex.com|vimeo.com|heute.at|ntv.com.tr|wargaming.net|metacritic.com|gq-magazine.co.uk|viafree.dk|viafree.fi|viafree.no|viafree.se|glamour.de|cbsnews.com|worldoftanks.ru|vg247.com|newyorker.com|games.miamiherald.com|eurogamer.es|eurogamer.de|bet.com|ikonpass.com|mcdelivery.de|id.trimble.com|skateboarding.transworld.net|newegg.com|docs.vmware.com|udemy.com|olx.pl|usmagazine.com|uefa.com|milwaukeetool.eu|muscleandfitness.com|20min.ch|ted.com|elisa.fi|nhl.com
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/138101
+@@||community.mozilla.org/wp-content/themes/mozilla/js/gdpr.js
+||community.mozilla.org/wp-content/themes/mozilla/js/gdpr.js$redirect=noopjs,important
+@@||community.mozilla.org/wp-content/uploads/$image
```

Comment:
Does it happens everywhere or only on `community.mozilla.org`?

Reply:
As far as i know, it happens only on the domain.

Reply:
In that case, please limit the rule to this site using `$domain=community.mozilla.org` and check how it works on the app.

Reply:
But, does it not enough to specify the domain by existing `community.mozilla.org` domain in the current rules?

Reply:
Looks like it's used only there, so yep, not necessary.